sys/arm64/vmm/vmm_arm64.c

```diff
@@ -1305,15 +1305,17 @@ vmmops_getreg(void *vcpui, int reg, uint64_t *retval)
 	regp = hypctx_regptr(hypctx, reg);
 	if (regp == NULL)
 		return (EINVAL);
-
-	*retval = *regp;
+	if (reg == VM_REG_GUEST_CPSR)
```

Comment:
What about the various CPSR, SCTLR, TTBR and TCR registers? vmmops_setreg handles them.